specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/disk.json

```diff
@@ -0,0 +1,1277 @@
+          "type": "string",
+          "description": "The uri to fetch the next page of disks. Call ListNext() with this to fetch the next page of disks."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List Disks operation response."
+    },
+    "DiskSku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": [
+            "Standard_LRS",
+            "Premium_LRS"
+          ],
+          "x-ms-enum": {
+            "name": "StorageAccountTypes",
+            "modelAsString": true
+          },
+          "description": "The sku name."
+        },
+        "tier": {
+          "type": "string",
+          "readOnly": true,
+          "default": "Standard",
+          "description": "The sku tier."
+        }
+      },
+      "description": "The disks sku name. Can be Standard_LRS or Premium_LRS."
+    },
+    "SnapshotSku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": [
+            "Standard_LRS",
+            "Premium_LRS",
+            "Standard_ZRS"
+          ],
+          "x-ms-enum": {
+            "name": "SnapshotStorageAccountTypes",
+            "modelAsString": true
+          },
+          "description": "The sku name."
+        },
+        "tier": {
+          "type": "string",
+          "readOnly": true,
+          "default": "Standard",
+          "description": "The sku tier."
+        }
+      },
+      "description": "The snapshots sku name. Can be Standard_LRS, Premium_LRS, or Standard_ZRS."
+    },
+    "DiskProperties": {
+      "properties": {
+        "timeCreated": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The time when the disk was created."
+        },
+        "osType": {
+          "type": "string",
+          "description": "The Operating System type.",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          }
+        },
+        "creationData": {
+          "$ref": "#/definitions/CreationData",
+          "description": "Disk source information. CreationData information cannot be changed after the disk has been created."
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "If creationData.createOption is Empty, this field is mandatory and it indicates the size of the VHD to create. If this field is present for updates or creation with other options, it indicates a resize. Resizes are only allowed if the disk is not attached to a running VM, and can only increase the disk's size."
+        },
+        "encryptionSettings": {
+          "$ref": "#/definitions/EncryptionSettings",
+          "description": "Encryption settings for disk or snapshot"
+        },
+        "provisioningState": {
```

Comment:
It would be great if you can make this enum and define all the supported values.

Reply:
this is also in compute.json, we will not change this unless crp changes it